manifests/opensearch-1.1.0.yml

```diff
@@ -0,0 +1,51 @@
+---
+schema-version: 1.0
+build:
+  name: OpenSearch
+  version: 1.1.0
+components:
+  - name: OpenSearch
+    repository: https://github.com/opensearch-project/OpenSearch.git
+    ref: 1.x
+  - name: common-utils
+    repository: https://github.com/opensearch-project/common-utils.git
+    ref: main
+  - name: job-scheduler
+    repository: https://github.com/opensearch-project/job-scheduler.git
+    ref: main
+  - name: sql
+    repository: https://github.com/opensearch-project/sql.git
+    ref: main
+  - name: alerting
+    repository: https://github.com/opensearch-project/alerting.git
+    ref: main
+  - name: notifications
+    repository: https://github.com/opensearch-project/notifications.git
+    ref: main
+  - name: security
+    repository: https://github.com/dblock/security.git
+    ref: fix-snapshot-build
+  # - name: performance-analyzer-rca
+  #   repository: https://github.com/opensearch-project/performance-analyzer-rca.git
+  #   ref: main
+  # - name: performance-analyzer
+  #   repository: https://github.com/opensearch-project/performance-analyzer.git
+  #   ref: main
+  - name: index-management
+    repository: https://github.com/opensearch-project/index-management.git
+    ref: main
+  - name: k-NN
+    repository: https://github.com/VijayanB/k-NN-2.git
```

Comment:
can we link issues to track for anything using a fork?

Reply:
Good idea, added.